docs/sources/getting-started/glossary.md

```diff
@@ -40,12 +40,36 @@ This topic lists words and abbreviations that are commonly used in the Grafana d
       Explore allows a user to focus on a building a query. User can refine the query to return the expected metrics before building a dashboard.
     </td>
   </tr>
+  <tr>
+    <td style="vertical-align: top">Export/Import</td>
+    <td>
+      Grafana includes the ability to export your dashboards to a file containing JSON. Community members sometimes share their created dashboards on the [Grafana Dashboards page](https://grafana.com/grafana/dashboards). Dashboards previously exported or found on this site may be imported by other users.
+    </td>
+  </tr>
+  <tr>
+    <td style="vertical-align: top">Exporter</td>
+    <td>
+      An exporter implements the translation of data that comes out of a data source into a format that Prometheus can digest.
+    </td>
+  </tr>
+  <tr>
+    <td style="vertical-align: top">Integration (Grafana Cloud)</td>
+    <td>
+      Each Integration in Grafana Cloud uses the cloud agent to connect your data source to Grafana Cloud for visualizing. Note: Prometheus uses the word “integrations” to refer to software that exposes Prometheus metrics without needing an exporter, which is a different use of the same word we use here.
```

Comment:
Why is Integration capitalized? Is this a product name?

Reply:
Yeah, that's how it is being used at the moment. Not my ideal, but it does differentiate Grafana Cloud Integrations from other uses of the same word.